Makefile

```
@@ -89,14 +89,31 @@ deps\:chglog:
 deps\:modules:
 	go mod download
 
+.PHONY: deps\:mock
+deps\:mock:
+	go install github.com/vektra/mockery/v2@v2.30.1
+
 .PHONY: deps
 deps:
 	@$(MAKE) deps:modules && \
 	$(MAKE) deps:bench && \
 	$(MAKE) deps:chglog && \
 	$(MAKE) deps:coverage && \
 	$(MAKE) deps:lint && \
-	$(MAKE) deps:test
+	$(MAKE) deps:test && \
+	$(MAKE) deps:mock
+
+.PHONY: mock
+mock:
+	$(MAKE) deps:mock
```

question: does this need a `@`.
```suggestion
	@$(MAKE) deps:mock
```

---

No strong preference, I can add one if you want

---

I think it caused a bug on macs, I forget the exact reason. Might as well

---

Added